recipes/reparation_blast/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "REPARATION_blast" %}
+{% set version = "v1.0.0" %}
+{% set sha256 = "74357b31c0c5929a97b61ab954a2809fa1e29b7f118baf32dd9d4807099b3f75" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/RickGelhausen/{{name}}/archive/{{version}}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [osx]
+
+requirements:
+  host:
+    - perl
+  run:
+    - perl
+    - perl-posix
+    - perl-bioperl
+    - samtools
+    - plastid
+    - r-ggplot2
+    - r-prroc
+    - r-rocr
+    - r-randomforest
+    - r-sizer
+    - prodigal
+    - glimmer
+    - blast >=2.7.1
+
+test:
+  commands:
+    - reparation.pl || true
```

Comment:
Doesn't this ``|| true`` mean the command will always return success - making it useless as a test?

If the tool doesn't have a ``--version`` or ``--help`` option or similar, can you use ``grep`` on the output at least to confirm the script is running and producing a meaningful output?

Reply:
Hi, thanks for the tip. 
You are right, this was a work-around for having no `--version` or `--help` option.
I have added a version option now.

Reply:
That's a better solution, and will be useful outside of BioConda too 👍